eng/common/maven.consume.settings.xml

```diff
@@ -0,0 +1,16 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" 
```

Comment:
Are these settings files consumed directly in the yml files? I'm not sure I like them sitting in the root of `eng/common` as they are only going to be interesting for people that publish to maven. At a min we should put them under a sub-dir of `eng/common` but we might also want to consider some opt-in model to syncing so that they only get sync'ed to the java and android repos. 

FYI @chidozieononiwu  is working on pulling out language specific stuff into the individual language repos instead of being in eng/common, but in this case these files are interesting for a subset of repo's and not all so maybe we should add some config somewhere to show what stuff to sync between the repo. Or we can use our common sync-directory templates to setup a direct sync of some directory between the java and android repo.